lib/src/rtc_peerconnection.dart

```diff
@@ -361,6 +383,116 @@ class RTCPeerConnection {
     return RTCDTMFSender(_peerConnectionId);
   }
 
+  /// Unified-Plan.
+  List<RTCRtpSender> get senders => _senders;
+
+  List<RTCRtpReceiver> get receivers => _receivers;
+
+  List<RTCRtpTransceiver> get transceivers => _transceivers;
+
+  Future<RTCRtpSender> createSender(String kind, String streamId) async {
+    try {
+      final response = await _channel.invokeMethod(
+          'createSender', <String, dynamic>{
+        'peerConnectionId': _peerConnectionId,
+        'kind': kind,
+        'streamId': streamId
+      });
+      var sender = RTCRtpSender.fromMap(response);
+      _senders.add(sender);
+      return sender;
+    } on PlatformException catch (e) {
+      throw 'Unable to RTCPeerConnection::createSender: ${e.message}';
+    }
+  }
+
+  Future<RTCRtpSender> addTrack(MediaStreamTrack track,
+      [List<String> streamIds]) async {
+    try {
+      final response =
+          await _channel.invokeMethod('addTrack', <String, dynamic>{
```

Comment:
addTrack missing on iOS currently?

Reply:
We are currently writing the Android native adaptation code, and we will write iOS code later, and we will eventually merge them all together.